src/utils/download.utils.ts

```diff
@@ -19,6 +34,7 @@ export function downloadFileToStr(urlStr: string): Promise<string> {
     });
   });
 }
+
 export function downloadFileToDestination(
   urlStr: string,
```

Comment:
change the api to support both, URl and string, and then you can drop the URL.toString()  in all the usages